shared/actions/favorite.js

```diff
@@ -229,9 +229,31 @@ function _notify(state: Types.FolderState): void {
   previousNotifyState = newNotifyState
 }
 
+// I think this can all go away, cc: @jzila
```

Comment:
Ah I misunderstood your message in Slack. Yeah we don't use these actions at all any more (since the widget refactor), so this is safe.